tests/py/test_teams.py

```diff
@@ -226,13 +225,10 @@ def test_casing_of_urls_survives(self):
         self.make_participant('alice', claimed_time='now', email_address='', last_paypal_result='')
         self.post_new(dict( self.valid_data
                           , homepage='Http://gratipay.com/'
-                          , onboarding_url='http://INSIDE.GRATipay.com/'
-                          , todo_url='hTTPS://github.com/GRATIPAY'
-                           ))
+                          , onboarding_url='http://INSIDE.GRATipay.com/'))
```

Comment:
Sorry to be picky but can we match the parentheses formatting here as it was before? :)